docs/content/usage/functions.en.md

```diff
@@ -38,6 +38,14 @@ $ fission route create --function hello --url /hello
 trigger '5327e9a7-6d87-4533-a4fb-c67f55b1e492' created
 ```
 
+Setup [FISSION_ROUTER](https://docs.fission.io/installation/env_vars/) environment variable if you haven't already.
+
+If you are using Minikube, it is fairly straightforward:
```

Comment:
The code snippet is useful. But consider the effort of maintenance in the future, it's better to redirect users to see the environment variable doc instead of having duplicate env set up code snippet here.